README.md

```diff
@@ -51,6 +51,10 @@ them to fit your particular use case.
     such as adding partitioning, clustering, metadata columns and prefixes to table names.
 *   [BigQuery Pipeline Utility](tools/bqpipeline) - Python utility class for
     defining data pipelines in BigQuery.
+*   [BigQuery Remote Function](examples/bq-remote-function) - It allows user to implement their custom
+    libraries or services in languages other than SQL or Javascript which are not part og UDFs.
+    The utility contains sample Java code to deploy cloud run gen2 instance and invoke the service 
```

Comment:
In a short sentence what does the example function do ?

Reply:
added!